src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -81,23 +202,55 @@ def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai, test_duratio
         ),
     )
 
-    # Always Validation
-    always_validation_sequence_set = [[]]
+    # Placement Always Validation
+    placement_always_validation_sequence_set = [[]]
 
-    # Eventually Validation
-    eventually_validation_sequence_set = [
+    # Placement Eventually Validation
+    placement_eventually_validation_sequence_set = [
         [
             # Ball should arrive within 5cm of placement point
-            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.05)]),
+            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
+            RobotEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
         ]
     ]
 
     simulated_test_runner.run_test(
-        eventually_validation_sequence_set=eventually_validation_sequence_set,
-        always_validation_sequence_set=always_validation_sequence_set,
+        eventually_validation_sequence_set=placement_eventually_validation_sequence_set,
+        always_validation_sequence_set=placement_always_validation_sequence_set,
         test_timeout_s=test_duration,
```

Comment:
Same as above, but would be nice to remove this param if we can (i.e set the timeout to a constant value as small as possible)